rust/src/http2/parser.rs

```diff
@@ -446,7 +446,7 @@ fn http2_parse_headers_block_literal_incindex<'a>(
                 //in case of overflow, best effort is to keep first headers
                 if dyn_headers.overflow > 0 {
                     if dyn_headers.overflow == 1 {
-                        if dyn_headers.current_size <= (HTTP2_MAX_TABLESIZE as usize) {
+                        if dyn_headers.current_size <= (unsafe { HTTP2_MAX_TABLESIZE } as usize) {
```

Comment:
@jasonish is there a better way ? The value is constant after it has been defined from the yaml configuration...

Reply:
No, not really.  There is stuff like `lazy_static`, but even that requires wrapping it in a mutex.

Here we can easily audit thats its only written to during startup state, and only read during runtime. So easy to validate, and about as clean as it gets.

The real fix would be some configuration object that is passed everytime a new state needs to be created (or even a state factory) that is registered somewhere. But our app-layer doesn't really have the facility for this.

Reply:
So, keeping this way then, thanks Jason